src/main/java/codeu/model/data/Activity.java

```diff
@@ -107,4 +107,10 @@ public String getThumbnail() {
   public void setIsPublic(Boolean isPublic) {
     this.isPublic = isPublic;
   }
+
+  @Override
+  public int compareTo(Activity o) {
+    Instant secondTime = o.getCreationTime();
+    return (secondTime.compareTo(creation));
+  }
```

Comment:
Not sure if this is why, but when I run the program it starts the activity feed from the oldest activity, not the most recent... Any thoughts on why this is happening now?

Reply:
Oh I saw the problem. I fixed it on my latest commit. Thanks, man! (Try it)

Reply:
Great, it works!